sdk/avs/Azure.ResourceManager.Avs/src/Generated/Models/AvsManagementCluster.cs

```diff
@@ -11,21 +11,72 @@
 namespace Azure.ResourceManager.Avs.Models
 {
     /// <summary> The properties of a management cluster. </summary>
-    public partial class AvsManagementCluster : CommonClusterProperties
+    public partial class AvsManagementCluster
```

Comment:
we need a customization partial class of this class to add back the base type, such as:
```csharp
public partial class AvsManagementCluster : CommonClusterProperties
{}
```
and then regenerate the code, you will see the duplicated properties in this type will disappear (ClusterSize, Hosts, etc)

Reply:
So after you regenerate, you might notice that `VsanDatastoreName` is not serialized in this type.
Therefore we need a method in the customized partial class:
```csharp
public partial class AvsManagementCluster : CommonClusterProperties
{
        /// <param name="writer"> The JSON writer. </param>
        /// <param name="options"> The client options for reading and writing models. </param>
        protected virtual void JsonModelWriteCore(Utf8JsonWriter writer, ModelReaderWriterOptions options)
        {
            base.JsonModelWriteCore(writer, options);
            if (Optional.IsDefined(VsanDatastoreName))
            {
                writer.WritePropertyName("vsanDatastoreName"u8);
                writer.WriteStringValue(VsanDatastoreName);
            }
        }
}
```
this is yet another issue in our current generator, I will create an issue to track it later. But for now we need this customized method to ensure the serialization will write this property.